src/main/java/com/fakng/fakngagrgtr/parser/amazon/AmazonParser.java

```diff
@@ -101,7 +102,12 @@ private String generateFullDescription(VacancyDTO dto) {
     }
 
     private ResponseDTO getPage(int offset) {
-        ResponseSpec request = getRequest(String.format(url, offset));
+        ResponseSpec request = getRequest(uriBuilder -> uriBuilder
```

Comment:
Why do we need this? Any practical usefulness?

Reply:
Main purpose is to increase code readability and making requests more flexible.

There is no logical connection between code and config variable. Also config variable should have implicitly reasoned format specifier when shouldn't. It degrades readability both the config file and parser's code. Also there is no flexibility with hardcoded url string.

With suggested approach if we need make changes in request params. We need ADD new param in the config file (not modify existing one).

#### For comparison:

In parser code:
https://github.com/fakng-agrgtr/fakng-agrgtr/blob/569e70e7cea332d1e40ed1c3bf8415f1a67baf88/src/main/java/com/fakng/fakngagrgtr/parser/google/GoogleParser.java#L94

https://github.com/fakng-agrgtr/fakng-agrgtr/blob/31ef5bee88c3733d696041e6d1e418a36d10a35d/src/main/java/com/fakng/fakngagrgtr/parser/google/GoogleParser.java#L93-L98

In application.yml file:
https://github.com/fakng-agrgtr/fakng-agrgtr/blob/569e70e7cea332d1e40ed1c3bf8415f1a67baf88/src/main/resources/application.yaml#L16-L18

https://github.com/fakng-agrgtr/fakng-agrgtr/blob/31ef5bee88c3733d696041e6d1e418a36d10a35d/src/main/resources/application.yaml#L16-L19

Reply:
One string is definitely better than 6

Nobody reads config values

Reply:
If we leave it like this, then I would put the lambda query creation in a separate method